x-pack/metricbeat/module/azure/monitor/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- name: monitor
+  type: group
+  description: >
+    monitor
+  fields:
```

Comment:
Having the template definition here would be very interesting to see how you are thinking about the metrics part. Everything float as a dynamic type?

Reply:
updated the template, float as dynamic type for the metric values